### PR TITLE
Completed HTML Links

### PR DESCRIPTION
**Pull request by louiewhitz**
https://user-images.githubusercontent.com/99847558/186032744-359b9b95-a875-4eb8-9df9-945dbbe5ed8d.mov

### CODE REVIEW

File: html-links/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Links</title>
+</head>
+<body>
+  <h1>Popular Search Engines</h1>
+  <ul>
+    <li><a href="https://www.google.com/">Google</a></li>
+
```
Comment (BrettAlbright) [inline html-links/index.html]: make sure to remove the whitespace from between the elements. you don't want there to be extra spaces like that.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Great work, small update!

File: html-links/index.html (hunk 1 above)
Comment (louiewhitz) [reply html-links/index.html]: Absolutely 👍

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**